Debian/bin/lib/Cyrus/Docker/Command/test.pm

```diff
@@ -0,0 +1,59 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::test;
+use Cyrus::Docker -command;
+
+use Path::Tiny;
+use Process::Status;
+
+sub execute ($self, $opt, $args) {
+  unless (-e '/run/rsyslogd.pid') {
+    system('/usr/sbin/rsyslogd');
+    Process::Status->assert_ok('starting rsyslog');
```

Comment:
s/starting/started/

Reply:
We discussed this, the text here is correct, as it's used to generate "starting rsyslog exited 12"